doc/flame/collision_detection.md

```diff
@@ -1,137 +1,143 @@
 # Collision detection
-If you want to have a full-blown physics engine in your game we recommend that you use
-Forge2D by adding [flame_forge2d](https://github.com/flame-engine/flame_forge2d) as a dependency.
-But if you have a simpler use-case and just want to check for collisions of components and improve
-the accuracy of gestures, Flame's built-in collision detection will serve you very well.
 
-If you have the following needs you should at least consider to use
-[Forge2D](https://github.com/flame-engine/forge2d):
- - Interacting realistic forces
- - Particle systems that can interact with other bodies
- - Joints between bodies
- - Many bodies at the same time (~50+ depends a bit on platform)
+Collision detection is needed in most games to detect and act upon two components intersecting each
+other. For example an arrow hitting an enemy or the player picking up a coin.
 
-It is a good idea to just use the Flame collision detection system if you on the other hand only
-need some of the following things (since it is slightly simpler to not involve Forge2D):
- - The ability to act on when some of your components collide
- - The ability to act on when your components collide with the screen boundaries
- - Complex shapes to act as a hitbox for your component so that gestures will be more accurate
- - Hitboxes that can tell what part of a component that collided with something
+In most collision detection systems you use something called hitboxes to create more precise
+bounding boxes of your components. In Flame the hitboxes are areas of the component that can react
+to collisions (and make [gesture input](inputs/gesture-input.md#GestureHitboxes) more accurate.
```

Comment:
```suggestion
to collisions.
```
The mentions of gestures distracts from the main idea of the document here (the reader now feels like they need to go and read what "gesture input" is).

Reply:
I think it is important to give the connection to the reader here though, and it's just a parenthesis.